app/api/entities/metadataValidators.js

```diff
@@ -57,7 +57,8 @@ const validateRequiredProperty = (property, value) => {
 const isValidLinkField = value =>
   isString(value.label) && value.label && isString(value.url) && value.url;
 
-const validateNumericProperty = value => isNumber(value) || value === '';
+const validateNumericProperty = value =>
+  isNumber(value) || value === '' || (isString(value) && `${parseInt(value, 10)}` === value);
```

Comment:
I am not really sure why this is needed.  Why would we want to allow strings that hold numbers?  This will force us to sanitize this data either upon save, or everywhere else where you want to do operations with these values.  Can you please elaborate on the reasoning behind this?

Thanks.

Reply:
This is the current state in planinternational. Not having this breaks any save action there.

Reply:
Yes, because CSV import treated everything as string.

I am concerned to end up with mixed data of strings and, moving forward, numbers.  Perhaps the correct approach is to do a migration that every numeric field is sanitized to a number?

@txau  ?

Reply:
We could merge it like this and do the migration further down the road and remove this validation to make it stricter.  That is another possible route.

Reply:
I would prefer to not pile another migration onto metadata-object :)